docs/config.toml

```diff
@@ -26,29 +26,31 @@ disableNavChevron = false # set true to hide next/prev chevron, default is false
 highlightClientSide = false # set true to use highlight.pack.js instead of the default hugo chroma highlighter
 menushortcutsnewtab = true # set true to open shortcuts links to a new tab/window
 enableGitInfo = true
-operatorVersion = "5.1.0"
-operatorVersionLatestRel5_0 = "5.0.5"
-imageCrunchyPostgres   = "registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-14.2-1"
-imageCrunchyPostgresPrivate       = "registry.crunchydata.com/crunchydata/crunchy-postgres:ubi8-14.2-1"
-imageCrunchyPGBackrest = "registry.developers.crunchydata.com/crunchydata/crunchy-pgbackrest:ubi8-2.38-0"
-imageCrunchyPGBackrestPrivate     = "registry.crunchydata.com/crunchydata/crunchy-pgbackrest:ubi8-2.38-0"
-imageCrunchyPGBouncer  = "registry.developers.crunchydata.com/crunchydata/crunchy-pgbouncer:ubi8-1.16-2"
-imageCrunchyExporter   = "registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:ubi8-5.1.0-0"
-imageCrunchyPGAdmin    = "registry.developers.crunchydata.com/crunchydata/crunchy-pgadmin4:ubi8-4.30-0"
+operatorVersion = "5.1.1"
+operatorVersionLatestRel5_0 = "5.0.6"
+imageCrunchyPostgres   = "registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-14.3-0"
+imageCrunchyPostgresPrivate       = "registry.crunchydata.com/crunchydata/crunchy-postgres:ubi8-14.3-0"
+imageCrunchyPGBackrest = "registry.developers.crunchydata.com/crunchydata/crunchy-pgbackrest:ubi8-2.38-1"
+imageCrunchyPGBackrestPrivate     = "registry.crunchydata.com/crunchydata/crunchy-pgbackrest:ubi8-2.38-1"
+imageCrunchyPGBouncer  = "registry.developers.crunchydata.com/crunchydata/crunchy-pgbouncer:ubi8-1.16-3"
+imageCrunchyExporter   = "registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:ubi8-5.1.1-0"
+imageCrunchyPGAdmin    = "registry.developers.crunchydata.com/crunchydata/crunchy-pgadmin4:ubi8-4.30-1"
 operatorRepository     = "registry.developers.crunchydata.com/crunchydata/postgres-operator"
```

Comment:
📝 I added a line after `imageCrunchyPGAdmin` here that's causing a conflict.

https://github.com/CrunchyData/postgres-operator/blob/b9e8c53e711e479062cbb558d8af5f750f19efe8/docs/config.toml#L37-L39

Reply:
oh i missed the -1 on that one thanks i'll update it now

Reply:
@cbandy I looked in the tagging sheet and for pgadmin v5.1.1 release pgadmin is ubi8-4.30-1